### PR TITLE
[Docs](bangc-ops): update v0.11.0 docs

### PR DESCRIPTION
**Pull request by duzekunKTH**
Thanks for your contribution and we appreciate it a lot. :rocket::rocket:

## 1. Motivation

更新 v0.11.0 版本的文档。

## 2. Modification

	modified:   .github/workflows/bangc_all_system_ci.yaml
	modified:   .github/workflows/bangc_ci.yaml
	modified:   .github/workflows/daily.yaml
	modified:   README.md
	modified:   bangc-ops/mlu_op.h
	modified:   build.property
	modified:   docs/bangc-docs/api_guide/update.rst
	modified:   docs/bangc-docs/release_notes/bangc_ops.rst
	modified:   docs/bangc-docs/user_guide/2_update_history/index.rst

### CODE REVIEW

File: docs/bangc-docs/release_notes/bangc_ops.rst [hunk 1]
```diff
@@ -71,6 +75,27 @@ Cambricon BANG C OPS具有以下特点：
    | Cambricon BANG C OPS v0.2.z | x86_64                 | MLU270、MLU290、MLU370         |
    +-----------------------------+------------------------+--------------------------------+
 
+v0.11.0
+-----------------
+
+特性变更
+~~~~~~~~~~~~~~~~~~~~~
+
+- 无
```
Comment (AquamarineSu) [inline docs/bangc-docs/release_notes/bangc_ops.rst:84]: 加个标点哦~ 

```suggestion
- 无。
```

File: docs/bangc-docs/release_notes/bangc_ops.rst [hunk 2]
```diff
@@ -71,6 +75,27 @@ Cambricon BANG C OPS具有以下特点：
...
+
+已修复问题
+~~~~~~~~~~~~~~~~~~~~~
+
+- 修复以下算子问题：
+
+   * 修复算子 yolo_box 防呆不完整问题。
```
Comment (AquamarineSu) [inline docs/bangc-docs/release_notes/bangc_ops.rst:91]:
```suggestion
   * 修复算子 ``yolo_box`` 防呆不完整问题。
```

**Review by AquamarineSu — APPROVED** [review]

File: docs/bangc-docs/release_notes/bangc_ops.rst [hunk 3]
```diff
@@ -71,6 +75,27 @@ Cambricon BANG C OPS具有以下特点：
...
+~~~~~~~~~~~~~~~~~~~~~
+
+- 修复以下算子问题：
+
+   * 修复算子 ``yolo_box`` 防呆不完整问题。
+
+已知遗留问题
+~~~~~~~~~~~~~~~~~~~~~
+
+- 无
```
Comment (duzekunKTH) [inline docs/bangc-docs/release_notes/bangc_ops.rst:96]:
```suggestion
- 无。
```

File: build.property [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
   "version": "0.10.0-1",
-  "build_requires": {"cntoolkit": ["release","3.7.0-1"], "cnnl":["release","1.21.1-1"]},
+  "build_requires": {"cntoolkit": ["release","3.8.1-1"], "cnnl":["release","1.22.0-1"]},
```
Comment (duzekunKTH) [inline build.property:3]:
```suggestion
  "build_requires": {"cntoolkit": ["release","3.7.0-1"], "cnnl":["release","1.21.1-1"]},
```

File: README.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ MLU-OPS 提供了以下功能：
 - 操作系统：
   - 支持 x86_64 下 Ubuntu18.04、Ubuntu20.04、Centos7.6、Centos8.5、Kylin10
 - 寒武纪 MLU SDK：
-  - 编译和运行时依赖 CNToolkit v3.7.0 或更高版本，CNNL v1.21.1 或者更高版本
+  - 编译和运行时依赖 CNToolkit v3.8.1 或更高版本，CNNL v1.22.0 或者更高版本
```
Comment (duzekunKTH) [inline README.md:51]:
```suggestion
  - 编译和运行时依赖 CNToolkit v3.7.0 或更高版本，CNNL v1.21.1 或者更高版本
```

File: .github/workflows/daily.yaml [hunk 6]
```diff
@@ -12,9 +12,9 @@ jobs:
     strategy:
       matrix:
         runner: [mlu370-m8]
-        mlu_ops_version : [0.10.0]
-        cntoolkit_version : [3.7.0]
-        cnnl_version: [1.21.1]
+        mlu_ops_version : [0.11.0]
+        cntoolkit_version : [3.8.1]
+        cnnl_version: [1.22.0]
```
Comment (duzekunKTH) [inline .github/workflows/daily.yaml:17]:
```suggestion
        cntoolkit_version : [3.7.0]
        cnnl_version: [1.21.1]
```

File: .github/workflows/bangc_all_system_ci.yaml [hunk 7]
```diff
@@ -18,9 +18,9 @@ jobs:
     strategy:
       matrix:
         runner: [mlu370-m8]
-        mlu_ops_version : [0.10.0]
-        cntoolkit_version : [3.7.0]
-        cnnl_version: [1.21.1]
+        mlu_ops_version : [0.11.0]
+        cntoolkit_version : [3.8.1]
+        cnnl_version: [1.22.0]
```
Comment (duzekunKTH) [inline .github/workflows/bangc_all_system_ci.yaml:23]:
```suggestion
        cntoolkit_version : [3.7.0]
        cnnl_version: [1.21.1]
```

File: docs/bangc-docs/release_notes/bangc_ops.rst [hunk 8]
```diff
@@ -20,6 +20,8 @@ Cambricon BANG C OPS具有以下特点：
    +-----------------------------+-----------------------------+
    | Cambricon BANG C OPS 版本   | 依赖组件版本                |
    +=============================+=============================+
+   | Cambricon BANG C OPS v0.11.z| CNToolkit >= v3.8.1         |
```
Comment (duzekunKTH) [inline docs/bangc-docs/release_notes/bangc_ops.rst:23]:
```suggestion
   +=============================+=============================+
   | Cambricon BANG C OPS v0.11.z| CNToolkit >= v3.7.0         |
```

**Review by Unireverse — APPROVED** [review]

**Review by PetrelYy — APPROVED** [review]

File: docs/bangc-docs/release_notes/bangc_ops.rst [hunk 9]
```diff
@@ -71,6 +75,27 @@ Cambricon BANG C OPS具有以下特点：
    | Cambricon BANG C OPS v0.2.z | x86_64                 | MLU270、MLU290、MLU370         |
    +-----------------------------+------------------------+--------------------------------+
 
+v0.11.0
+-----------------
+
+特性变更
+~~~~~~~~~~~~~~~~~~~~~
+
+- 无。
```
Comment (duzekunKTH) [inline docs/bangc-docs/release_notes/bangc_ops.rst:84]:
```suggestion
- 新增底层依赖 CNNL。
```

File: docs/bangc-docs/release_notes/bangc_ops.rst [hunk 10]
```diff
@@ -20,6 +20,8 @@ Cambricon BANG C OPS具有以下特点：
    +-----------------------------+-----------------------------+
    | Cambricon BANG C OPS 版本   | 依赖组件版本                |
    +=============================+=============================+
+   | Cambricon BANG C OPS v0.11.z| CNToolkit >= v3.7.0         |
```
Comment (DanieeelLiu) [inline docs/bangc-docs/release_notes/bangc_ops.rst:23]: cnnl 也添加下？

**Pull request merged**